### DISCUSSION TITLE
Module 1.1 Trouble with main.py - SD Card failed to initialize

### DISCUSSION THREAD

**Discussion by jp-lozano** [AccelerationConsortium/ac-microcourses#23] [Hello World] [answered]
Hello, I have been trying to get main.py to work but up to now it seems to continue having issues. It was originally disconnecting so I re-downloaded the Demo package. The light is on at GP9 and GP15, but now presents a message like the one shown below (SD card failed to initialize). 

<img width="595" alt="Screenshot 2024-07-17 at 12 16 06" src="https://github.com/user-attachments/assets/a02a6ce6-789c-44aa-a3fa-6fd18717ff0b">

**Comment by sgbaird** [comment, marked as answer]
Hi @jp-lozano, thanks for this. The error message can be safely ignored. If an SD card is present, it is used as a backup data log, otherwise it skips it. As long as you see "Waiting for experiment requests..." it should be good to go.

Also surfacing the following here for others that might have WiFi connection issues:

![image](https://github.com/user-attachments/assets/bf39ebc6-2684-4bea-bb73-b616149d8406)
